packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -255,14 +297,41 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
           this.forceUpdate();
         }
 
+        document.addEventListener('keydown', this._onKeyDown, true);
+
         // We dont want to the user to immediatley trigger the coachmark when it's opened
         this._async.setTimeout(() => {
           this._addProximityHandler(this.props.mouseProximityOffset);
         }, this.props.delayBeforeMouseOpen!);
+
+        if (this.props.ariaAlertText) {
+          this._async.setTimeout(() => {
+            if (this.props.ariaAlertText && this._ariaAlertContainer.current) {
+              const alertText = document.createTextNode(this.props.ariaAlertText);
```

Comment:
Why are you creating a new node instead of doing something like `setState({ alertText: this.props.alertText})` and then in the container you can populate the alert text. That should work the same I believe.

Reply:
Fixed in latest commit